seed/filtersets.py

```diff
@@ -88,7 +91,7 @@ def year_filter(self, queryset, name, value):
         name = "{} Calendar Year".format(value)
         cycles = queryset.filter(name__contains=name)
         if not cycles:
-            start = datetime(int(value), 1, 1)
+            start = make_aware(datetime(int(value), 1, 1), pytz.UTC)
```

Comment:
Is this changing a default then?  Or is this just being intentional about what it was defaulting to before?